internal/services/keyvault/key_vault_managed_hardware_security_module_resource_test.go

```diff
@@ -23,14 +23,14 @@ func TestAccKeyVaultManagedHardwareSecurityModule(t *testing.T) {
 			"basic": testAccDataSourceKeyVaultManagedHardwareSecurityModule_basic,
 		},
 		"resource": {
-			"basic":    testAccKeyVaultManagedHardwareSecurityModule_basic,
+			"basic":    TestAccKeyVaultManagedHardwareSecurityModule_basic,
```

Comment:
Was this case change for local testing?  This will run twice in the test suite I think.
```suggestion
			"basic":    testAccKeyVaultManagedHardwareSecurityModule_basic,
```

Reply:
🤦 yep! Thank you thank you!